ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -xeuo pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+  echo "Try to login..."
+  "${IBMCLOUD_CLI}" login -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function checkCli() {
+  export IBMCLOUD_CLI=ibmcloud
+  export IBMCLOUD_HOME=/output
+  echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
+  echo "Pause for debug...."
+  command -v ${IBMCLOUD_CLI}
+  ${IBMCLOUD_CLI} --version
+  ${IBMCLOUD_CLI} plugin list
+}
+
+checkCli
```

Comment:
I do not think need this check, when we always set `set -o errexit`. And the step is limited to use "upi-installer:4.12" to run the step.

Reply:
>   export IBMCLOUD_CLI=ibmcloud
>   export IBMCLOUD_HOME=/output

Reply:
How about update as the following: 
# IBM Cloud CLI login
function ibmcloud_login {
  export IBMCLOUD_CLI=ibmcloud
  export IBMCLOUD_HOME=/output
  command -v ${IBMCLOUD_CLI}
  echo "Try to login..."
  "${IBMCLOUD_CLI}" login -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
  region="${LEASED_RESOURCE}"
  export region
}

ibmcloud_login

Reply:
Is `export IBMCLOUD_HOME=/outpu` line a MUST?

I suggest not to place the lines in a function which is only called one time in the step, but directly run the lines.

Reply:
hided IBMCLOUD_HOME=/output, fail to use the plugin
[build-log.txt](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/39088/rehearse-39088-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-ibmcloud-ipi-workers-rhel8-p2-f14/1656510467041923072/artifacts/ibmcloud-ipi-workers-rhel8-p2-f14/ibmcloud-provision-vpc/build-log.txt)

> Create VPC...
FAILED
'is' is not a registered command. Check your list of installed plug-ins. See 'ibmcloud help'.